src/helm/benchmark/presentation/schema.py

```diff
@@ -214,6 +214,11 @@ class RunGroup(Field):
     # TODO: remove when we don't want helm-summarize to support runs before November 2023 anymore.
     adapter_keys_shown: List[str] = field(default_factory=lambda: ["model_deployment", "model"])
 
+    # Whether to accept non registered models or not.
+    # Used for huggingface models, a default ModelMetadata will be used when missing,
+    # Otherwise, if this is False, raises an error.
+    enable_huggingface_models: bool = True
```

Comment:
Change this to `allow_unknown_models` or something like that.

Update `schema.yaml` to set this to `true`.